setup.cfg

```
@@ -47,6 +47,7 @@ install_requires =
     PyYAML<4;python_version<"2.7"
     wheel<=0.29.0;python_version<"2.7"
     compatibleversion>=0.1.2
+    oschmod
```

To ensure we remain license-compatible...
```suggestion
    oschmod>=0.1.3
```